backend/frontend/public/locales/en/phonenumberloginscreen.json

```diff
@@ -0,0 +1,22 @@
+{
+    "loginUser": "Login User",
+    "loginDescription": "Log in right away for our advanced sports software solutions, created to address issues in regular sporting events and activities.",
+    "getStartedWithStrongr": "Get Started With Strongr",
```

Comment:
Not a blocker, but I would suggest to use dynamic variable for Strongr (not only here, wherever we are using), so that incase of future changes to app name, will be easy.

Reply:
Sure,will change it.